resolvers/bodyMeasurement/bodyMeasurementMutations.js

```diff
@@ -11,7 +11,20 @@ const addBodyMeasurement = async(root, args, context, info)=>
 const deleteBodyMeasurement = async(root, args, context, info)=>
     await context.prisma.deleteBodyMeasurement({id: args.id});
 
+const editBodyMeasurement = async(root, args, context, info)=>
+    await context.prisma.updateBodyMeasurement({
+        data: {
+            hips: args.hips,
+            waist: args.waist,
+            rightArm: args.rightArm,
+            leftArm: args.leftArm,
+            leftLeg: args.leftLeg,
```

Comment:
what about gastroc measurements?
Pectoral penation angle?
What is this amateur hour? 
I need all my stats!